src/main/app/src/app/klanten/persoonsgegevens/persoonsgegevens.component.html

```diff
@@ -35,7 +35,7 @@
                         <mat-divider></mat-divider>
                     </div>
                     <zac-static-text [label]="'geboortedatum' | translate"
-                                     [value]="persoon?.geboortedatum"></zac-static-text>
+                                     [value]="persoon?.geboortedatum | datum | empty"></zac-static-text>
```

Comment:
Waarom heb je die empty erachter gezet? Wat doet die empty?

Reply:
Zodat er een streepje getoond wordt als de waarde niet geset is. Omdat de variabele `persoon` niet geset zou kunnen zijn (adhv de `?.`) leek het me verstandig om wel iets nuttigs te tonen in plaats van een leeg veld.

Misschien begrijp ik verkeerd wat de `empty`-pipe precies doet. Klopt het dat deze in het geval van een ongeldige of niet-bestaande waarde een liggend streepje toont?

Reply:
Heb je gekeken hoe het veld gevuld wordt? Kan het wel leeg zijn?

Reply:
Ah, volgens mij begrijp ik het. Tijdens het laden van de informatie is de `persoon` `null`. Pas als alle informatie geladen is wordt de boel getoond. De `?.` wordt alleen gebruikt om fouten te voorkomen voordat de Observable een waarde geemit heeft. De `empty` pipe is overbodig, omdat de informatie toch nooit getoond wordt aan de gebruiker tot de `persoon` is opgehaald. 

Ik heb een PR https://github.com/NL-AMS-LOCGOV/zaakafhandelcomponent/pull/1471 aangemaakt om de overbodige empty-pipe te verwijderen.